go.mod

```diff
@@ -23,6 +23,7 @@ require (
 	github.com/urfave/negroni v1.0.0
 	google.golang.org/grpc v1.25.1
 	gopkg.in/DATA-DOG/go-sqlmock.v1 v1.3.0 // indirect
+	gopkg.in/yaml.v2 v2.2.4
```

Comment:
I am curious as why you didn't choose to use `sigs.k8s.io/yaml` instead?

Reply:
Well … I just typed away and the import was added automatically when I saved the file. ^^

Reply:
I'm not one to judge :)